### DISCUSSION TITLE
Migration cripled mailscanner

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#15075] [How to]
Migrated to last stable and got this:

2021-08-15 11:26:02.3134 [warning] - Unable to turn on scanning because cron is currently active.
 - 0 modules/OSSMailScanner/models/Record.php (532)  >>  App\Log->warning('Unable to turn on scanning because cron is currently active.')
 - 1 modules/OSSMailScanner/crons/Action.php (27)  >>  OSSMailScanner_Record_Model->executeCron('apache2handler - ')
 - 2 cron.php (99)  >>  OSSMailScanner_Action_Cron->process()

Anybody got clue?

**Comment by rskrzypczak** [comment]
I don't see any connection between migration and the problem you described.
However, you can go to the panel below and stop the current action.
Presumably some e-mail is causing problems.
![image](https://user-images.githubusercontent.com/10221999/129514248-11ede645-abfd-45ea-bced-2d44f3e4b998.png)
